src/Plugin/Field/FieldWidget/ColorSchemeWidget.php

```diff
@@ -103,21 +104,17 @@ public function formElement(FieldItemListInterface $items, $delta, array $elemen
   protected function getColorSchemeOptions(): array {
     $theme_name = $this->themeHandler->getDefault();
     $theme_info = $this->themeExtensionList->getExtensionInfo($theme_name);
+    $options = [];
 
-    if (!empty($theme_info['color_scheme'])) {
-      // Parse the options to be more humanly readable.
-      $keys = $theme_info['color_scheme'];
-
-      $values = array_map(function ($key) {
-        $name = str_replace('_', ' ', $key);
-
-        return ucfirst($name);
-      }, $keys);
+    if (!isset($theme_info['color_scheme'])) {
+      return $options;
+    }
 
-      return array_combine($keys, $values);
+    foreach ($theme_info['color_scheme'] as $key => $label) {
+      $options[Html::getClass($key)] = strip_tags($label);
```

Comment:
Is this actually needed? In the end, we store the value in a string, and there's no need to transform it for the options field. Maybe we should enforce validation when fetching the options, even adding a hook_requirements that checks the current theme for the values.

Reply:
Any validation should be done in the requirements hook then.
I'm not convinced how validation is better in this case than processing, seems rather that we pass the cat from one to another.

Reply:
My main point is: do we need to have values in a specific format, eg no spaces, [, ], / (which is what Html::getClass does)?
If we don't, I would rather expect the values to be the ones declared, and not tamper with them.
If we do, then I would set validation on them, and exclude the ones that don't respect the format, but still avoid changing the values.

Reply:
I've implemented the filtering of options